website/layouts/api.jsx

```diff
@@ -1,21 +1,42 @@
-import DocsPage, { getInitialProps } from '../components/docs-page'
-import orderData from '../data/api-navigation.js'
-import { frontMatter } from '../pages/api-docs/**/*.mdx'
+import DocsPage from '@hashicorp/react-docs-page'
+import order from '../data/api-navigation.js'
+import { frontMatter as data } from '../pages/api-docs/**/*.mdx'
+import Head from 'next/head'
+import Link from 'next/link'
 
 function ApiLayoutWrapper(pageMeta) {
   function ApiLayout(props) {
     return (
       <DocsPage
         {...props}
-        orderData={orderData}
-        frontMatter={frontMatter}
-        category="api-docs"
-        pageMeta={pageMeta}
+        product="nomad"
+        head={{
+          is: Head,
+          title: `${pageMeta.page_title} | Nomad by HashiCorp`,
+          description: pageMeta.description,
+          siteName: 'Nomad by HashiCorp'
+        }}
+        sidenav={{
+          Link,
+          category: 'docs',
+          currentPage: props.path,
+          data,
+          order
+        }}
+        resourceURL={`https://github.com/hashicorp/nomad/blob/master/website/pages/${pageMeta.__resourcePath}`}
       />
     )
   }
+  /*
+    data,
+  order,
+  currentPage,
+  category,
+  Link,
+  product
+  */
 
-  ApiLayout.getInitialProps = getInitialProps
+  ApiLayout.getInitialProps = ({ asPath }) => ({ path: asPath })
```

Comment:
any reason this changed?

Reply:
Yes, this was moved here from the internal `DocsPage` component. It’s a `sidenav` prop.

Reply:
I guess I mean is there any reason we don't still want to bundle it with the component, as it was previously?